### PR TITLE
notify on Canton bumps

### PR DESCRIPTION
**Pull request by garyverhaegen-da**
CHANGELOG_BEGIN
CHANGELOG_END

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -15,7 +15,7 @@ azure-pipelines.yml @garyverhaegen-da
 azure-cron.yml      @garyverhaegen-da
 /release/           @garyverhaegen-da
 /release.sh         @garyverhaegen-da
-/canton_dep.bzl     @garyverhaegen-da
+/canton_dep.bzl     @garyverhaegen-da @remyhaemmerle-da
```
Comment (remyhaemmerle-da) [inline CODEOWNERS:18]: @rgugliel-da @phoebe-da Do you want to be codeowners as well ? 
Like that you will automatically added as reviewer for the automatic bump of canton in the main repo ?

Comment (remyhaemmerle-da) [reply CODEOWNERS:18]: like that you will be notify for each update of canton in the daml repo.

Comment (garyverhaegen-da) [reply CODEOWNERS:18]: And to clarify: in this repo, we use CODEOWNERS _only_ as a notification mechanism, this wouldn't give you any extra responsibility.

Comment (rgugliel-da) [reply CODEOWNERS:18]: Yes, that would be nice, thanks!

Comment (garyverhaegen-da) [reply CODEOWNERS]: Added. I'll give @phoebe-da some time to answer, but worst case we can make another PR later on.

**Review by remyhaemmerle-da — APPROVED** [review]
Thanks.

**Pull request merged**